src/SCS.jl

```diff
@@ -14,8 +12,6 @@ else
     error("SCS not properly installed. Please run Pkg.build(\"SCS\") and restart julia")
 end
 
-using Compat
-
 function __init__()
     vnum = VersionNumber(SCS_version())
     depsdir = realpath(joinpath(dirname(@__FILE__),"..","deps"))
```

Comment:
Below: we need version `v2.0.*`, the not-yet-released `v2.1.0` of scs has additional fields in `SCSSettings` so it's not compatible